.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+dist: trusty
```

Comment:
As @probonopd mentioned, it should run on almost anything post 2014.  I've only ever seen issues in cases like this when dealing with either 'enterprise' distros (where the user should just use the packaged version or build it themself locally), or tight embedded distros with really aggressive hardening like Alpine.